### DISCUSSION TITLE
Time zone doesn't reflect environment variable in Docker

### DISCUSSION THREAD

**Discussion by HawaiianPizza** [dani-garcia/vaultwarden#2343] [Q&A]
### Subject of the issue
Time zone doesn't reflect what is stated in the environment variable.

### Deployment environment

* vaultwarden version: 1.24.0

* Install method: Docker image

* Clients used: Web vault, Android

### Steps to reproduce

1. Inside docker-compose.yml, set the environment variable to:

```
environment:
      - TZ="America/Toronto"
```

2. Go to the Admin panel.
3. Go to Users.
4. `Created at` and `Last active` are only in UTC.

### Expected behaviour

Time zone is expected to display in the specified time zone in the docker-compose file.
### Actual behaviour

Time zone in docker-compose file is ignored and instead Vaultwarden uses UTC.

**Comment by BlackDex** [comment]
Works for me.
Within `docker-compose.yml` you need to remove the quotes `"` around the value.

**Reply by jjlin** [reply]
Quoting is an issue with `docker run --env-file ...` (and this aspect is mentioned at https://github.com/dani-garcia/vaultwarden/wiki/Configuration-overview), but I don't believe it's an issue with Docker Compose, which has its own implementation of env var parsing AFAICT.

Edit: Nevermind, quoting is an issue when using `environment`, but not when using `env_file` for some reason.

**Reply by HawaiianPizza** [reply]
@BlackDex Worked for me! The quotation marks were the culprit. Thank you!

**Comment by BlackDex** [comment]
I added a warning in the wiki regarding the quotes: https://github.com/dani-garcia/vaultwarden/wiki/Using-Docker-Compose

**Comment by jjlin** [comment]
Based on your steps, you didn't restart the service (e.g., via `docker-compose up -d`) after modifying the env var, so the change was not put into effect.

**Reply by BlackDex** [reply]
Even then, quoting that value will also cause it to not work at all.

**Reply by jjlin** [reply]
Hmm, yeah it seems if you use
```yaml
environment:
  - FOO="bar"
```
then the quotes are retained. But if you use
```yaml
env_file: file.env
```
where `file.env` contains `FOO="bar"`, then the quotes aren't retained.
